src/primitives/transaction.cpp

```diff
@@ -84,6 +84,17 @@ CTransaction &CTransaction::operator=(const CTransaction &tx)
     return *this;
 }
 
+bool CTransaction::IsEquivalentTo(const CTransaction &tx) const
+{
+    CMutableTransaction tx1 = *this;
```

Comment:
This function is basically a cut and paste from wallet.h/wallet.cpp.  You could easily  remove it from wallet.h/wallet.cpp because it inherits.

Reply:
Good catch, looks like this was done in https://github.com/bitcoin/bitcoin/pull/6365. I'll replace the commit introducing this with a revert commit.